src/main/java/uk/gov/hmcts/reform/unspec/model/ServedDocumentFiles.java

```diff
@@ -20,19 +20,35 @@
     private List<Element<DocumentWithRegex>> medicalReport;
     private List<Element<DocumentWithRegex>> scheduleOfLoss;
     private Document particularsOfClaimDocument;
+    private Document particularsOfClaimDocumentNew;
     private String particularsOfClaimText;
     private List<Element<DocumentWithRegex>> certificateOfSuitability;
 
     @JsonIgnore
     public List<String> getErrors() {
         List<String> errors = new ArrayList<>();
         if (ofNullable(particularsOfClaimDocument).isPresent() && ofNullable(particularsOfClaimText).isPresent()) {
```

Comment:
if this is new implementation, then it should use `particularsOfClaimDocumentNew` field